drivers/wifi/nrf700x/Kconfig.nrf700x

```diff
@@ -126,6 +126,7 @@ config NRF700X_MAX_TX_AGGREGATION
 
 config NRF700X_MAX_TX_TOKENS
 	int "Maximum number of TX tokens"
+	range 5 255 if !NRF700X_RADIO_TEST
```

Comment:
The range should only be till 12, we will not support more than that.

Reply:
sure. will fix